pkg/controllers/managementuser/rbac/globalrolebinding_handler.go

```diff
@@ -179,10 +182,32 @@ func (c *grbHandler) ensureProvisioningClusterAdminBinding(obj *v3.GlobalRoleBin
 	}
 
 	provCluster := pClusters[0]
-
 	subject := rbac.GetGRBSubject(obj)
-	rbName := name.SafeConcatName(rbac.ProvisioningClusterAdminName(provCluster), rbac.GetGRBTargetKey(obj))
 
+	// Add the restricted admin user as a member of the downstream cluster
+	// by creating a CRTB in the local custer in the namespace named after the downstream cluster.
+	_, err = c.clusterRoleTemplateBindings.Create(&v3.ClusterRoleTemplateBinding{
```

Comment:
Let's check the cache first to see if the CRTB exists fist and then only perform a create if it doesn't.